server.py

```diff
@@ -119,25 +120,31 @@ def data_loader(path, context):
         for content_fname in glob.glob(os.path.join(path, "*.*")):
             if (content_fname.endswith("yaml") or content_fname.endswith("json")):
                 continue
-            content_path = os.path.join(content_fname)
-            # print("Content file: {}, properties: {}".format(cf, str(context)))
-            entries.append(Image(content_path, context))
+            # csv files contain a list of paths to image files to be loaded
+            elif content_fname.endswith("csv"):
+                with open(os.path.join(content_fname), 'r') as list_file:
+                    for f in list_file:
+                        f = f.replace("\n", "")
```

Comment:
Lets rename `f` to `filename` or something. Replace this line with `f = f.strip()` that removes any whitespace (also `\r\n`) at the ends.